pkg/asset/manifests/node.go

```diff
@@ -0,0 +1,82 @@
+package manifests
+
+import (
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"path/filepath"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+)
+
+var (
+	nodeCfgFilename = filepath.Join(manifestDir, "cluster-node-02-config.yml")
+)
+
+// Node generates the cluster-node-*.yml files.
+type Node struct {
+	FileList []*asset.File
+}
+
+var _ asset.WritableAsset = (*Node)(nil)
+
+// Name returns a human friendly name for the asset.
+func (*Node) Name() string {
+	return "Node Config"
+}
+
+// Dependencies returns all of the dependencies directly needed to generate
+// the asset.
+func (*Node) Dependencies() []asset.Asset {
+	return []asset.Asset{
+		&installconfig.ClusterID{},
+		&installconfig.InstallConfig{},
+	}
+}
+
+// Generate generates the Node config and its CRD.
+func (n *Node) Generate(dependencies asset.Parents) error {
+	clusterID := &installconfig.ClusterID{}
+	installConfig := &installconfig.InstallConfig{}
+	dependencies.Get(clusterID, installConfig)
```

Comment:
It does not look you are using these dependencies.

Reply:
If we have to support the WorkerLatencyProfiles via the install-config (in future), the dependency may be required. Anyway, as of today, this is not really required.